crates/edr_evm/src/blockchain/local.rs

```diff
@@ -108,7 +105,8 @@ impl LocalBlockchain {
                 B64::from_limbs([66u64.to_be()])
             },
             base_fee: if spec_id >= SpecId::LONDON {
-                Some(base_fee.ok_or(CreationError::MissingBaseFee)?)
+                // Initial base fee from https://eips.ethereum.org/EIPS/eip-1559
+                Some(base_fee.unwrap_or(U256::from(1_000_000_000)))
```

Comment:
I think this is incorrect as this will now always default to `1_000_000_000` even if it's not the genesis block.

The external caller should pass in `1_000_000_000` for the genesis block instead

Reply:
Hmm, I must've misunderstood reading the code then. I thought this was just the header for the genesis block?

Reply:
It's for any locally created block